data/parse_smogon_stats.py

```diff
@@ -41,64 +41,25 @@ def get_smogon_stats_file_name(game_mode, month_delta=1):
-        pokemon_name = normalize_name(segments[1])
-        pokemon_information[pokemon_name] = dict()
-        pokemon_information[pokemon_name][SPREADS_STRING] = list()
-        pokemon_information[pokemon_name][ITEM_STRING] = list()
-        pokemon_information[pokemon_name][MOVES_STRING] = list()
-        pokemon_information[pokemon_name][ABILITY_STRING] = list()
-        for segment in it:
-            if normalize_name(segment) == SPREADS_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if ':' in segment:
-                        split_segment = segment.split()
-                        spread = split_segment[0]
-                        nature = normalize_name(spread.split(':')[0])
-                        evs = spread.split(':')[1].replace('/', ',')
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        pokemon_information[pokemon_name][SPREADS_STRING].append((nature, evs, percentage))
-
-            elif normalize_name(segment) == ITEM_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        item = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if item != OTHER_STRING:
-                            pokemon_information[pokemon_name][ITEM_STRING].append((item, percentage))
-
-            elif normalize_name(segment) == MOVES_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        move = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if move != OTHER_STRING:
-                            if constants.HIDDEN_POWER in move:
-                                move = "{}{}".format(move, constants.HIDDEN_POWER_ACTIVE_MOVE_BASE_DAMAGE_STRING)
-                            pokemon_information[pokemon_name][MOVES_STRING].append((move, percentage))
-
-            elif normalize_name(segment) == ABILITY_STRING:
-                while SECTION_END_STRING not in segment:
-                    segment = next(it)
-                    if '%' in segment:
-                        ability = normalize_name(re.sub(PERCENTAGES_REGEX, '', segment).strip())
-                        percentage = float(re.search(PERCENTAGES_REGEX, segment).group()[:-1])
-                        if ability != OTHER_STRING:
-                            pokemon_information[pokemon_name][ABILITY_STRING].append((ability, percentage))
-
-    return pokemon_information
+    infos = r.json()['data']
+    final_infos = {}
+    for x in infos.keys():
```

Comment:
The pokemon name _x_ is not normalized - this bot needs the name to be normalized. You can use the function:

```
from showdown.engine.helpers import normalize_name
```